core/components/molecules/sidesheet/__stories__/scrolling.story.jsx

```diff
@@ -0,0 +1,230 @@
+        Open Sidesheet
+      </Button>
+      <Sidesheet {...options}>
+        <Column className="py-7">
+          <Row>
+            <div className="w-100">
+              <Label required={true} withInput={true}>
+                Goal
+              </Label>
+              <Input placeholder="Placeholder" />
+            </div>
+            <div className="d-flex mt-6">
+              <div>
+                <Label required={true} withInput={true}>
+                  Achieve by
+                </Label>
+                <InputMask icon="event" mask={dateMask} validators={dateValidator} placeholder="mm/dd/yyyy" />
+              </div>
+              <div className="ml-6">
+                <Label required={true} withInput={true}>
+                  Priority
+                </Label>
+                <div className="d-flex py-3">
+                  <Radio className="mr-4" label="High" name="Priority" size="regular" value="High" />
+                  <Radio
+                    defaultChecked={true}
+                    className="mr-4"
+                    label="Medium"
+                    name="Priority"
+                    size="regular"
+                    value="Medium"
+                  />
+                  <Radio label="Low" name="Priority" size="regular" value="Low" />
+                </div>
+              </div>
+            </div>
+          </Row>
+          <Row className="mt-7">
+            <Label withInput={true} required={true}>
+              Care team interventions
+            </Label>
+            <Textarea
+              className="mt-4"
+              aria-labelledby="Textarea"
+              name="Textarea"
+              placeholder="Define intervention"
+              resize={true}
+              rows={5}
+            />
+            <Button className="mt-4" size="tiny" icon="refresh" iconAlign="left" appearance="transparent">
+              Add intervention
+            </Button>
+          </Row>
+          <Row className="mt-7">
+            <Label className="d-block w-100" withInput={true}>
+              Patient actions
+            </Label>
+            <Button className="mt-4" size="tiny" icon="refresh" iconAlign="left" appearance="transparent">
+              Add patient action
+            </Button>
+          </Row>
+          <Row className="pr-7 mt-7">
+            <Label withInput={true}>Motivational score</Label>
+            <Slider className="mt-6" defaultValue={2} label="Confidence" onChange={function () {}} stepSize={1} />
+            <Slider className="mt-8" defaultValue={2} label="Readiness" onChange={function () {}} stepSize={1} />
+            <Slider className="mt-8" defaultValue={2} label="Importance" onChange={function () {}} stepSize={1} />
+            <Slider className="mt-8" defaultValue={2} label="Punctuality" onChange={function () {}} stepSize={1} />
+            <Slider className="mt-8 mb-8" defaultValue={2} label="Clarity" onChange={function () {}} stepSize={1} />
```

Comment:
Please remove the `onChange` function if not required.

Reply:
Done